### PR TITLE
[BugFix] Fix broker load parquet file has uppercase column name

### LINKED ISSUE #9858: Broker load parquet file fail which has uppercase column
### Steps to reproduce the behavior (Required)
1. create table by spark sql
```
CREATE TABLE `default`.`tdd` (
  `k` STRING,
  `vVv` INT,
  `t` STRING)
USING parquet
PARTITIONED BY (t)
```
2. load data
```
LOAD LABEL xxx
(
    DATA INFILE('hdfs://x.x.x.x:9000/user/hive/warehouse/tdd/t=4/*')
    INTO TABLE tcc
    FORMAT AS "parquet"
    (k, vVv)
    COLUMNS FROM PATH AS (t)
    set(k=t, v=vVv)
)
WITH BROKER 'broker1'
PROPERTIES
(
    'timeout' = '3600'
)
```

### Expected behavior (Required)
load success

### Real behavior (Required)
load fail

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #9858

## Problem Summary(Required) ：

Parquet file create by spark sql will have uppercase column name, we can not direct covert to lowercase.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|2|2|100.00%|[]|

**Review by Astralidea — APPROVED** [review]

**Comment by meegoo** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java [hunk 1]
```diff
@@ -2478,12 +2478,12 @@ private DataDescription getDataDescription(StarRocksParser.DataDescContext conte
         List<String> colList = null;
         if (context.colList != null) {
             List<Identifier> identifiers = visit(context.colList.identifier(), Identifier.class);
-            colList = identifiers.stream().map(Identifier::getValue).map(String::toLowerCase).collect(toList());
+            colList = identifiers.stream().map(Identifier::getValue).collect(toList());
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java:2481]: Add a case to the StarRocksTest.

**Review by chaoyli — APPROVED** [review]

**Comment by meegoo** [comment]
@mergify backport branch-2.4

**Pull request merged**